pkg/common/outcomes/info.go

```diff
@@ -0,0 +1,25 @@
+package outcomes
+
+var _ Outcome = (*Info)(nil)
+
+// Info is the check outcome for checks that cannot be categorized as pass or fail, but instead simply give information
+// about the status of the mesh/its components. These checks show dynamically-generated diagnostic information to the user to provide more context.
+// Ex: check whether a pod participates in an SMI TrafficSplit or not, if yes - output the name of the TrafficSplit
+type Info struct {
```

Comment:
Nit:
"Info" could be "None" since there wasn't any clear outcome. Pass or fail outcomes also provide info about the status of the mesh so using "Info" might be confusing. Just an idea :)

Reply:
Hmm I'm not sure if "None" as an outcome type would be entirely accurate because we are still passing it an information string to be outputted but with "none" I imagine literally a blank outcome. I'm thinking of it more to align with like log levels error/warn/trace/**info**

Reply:
I agree, could see that working well for readability! but not sure if we should diverge between the command line output and the outcome type in code